.pre-commit-config.yaml

```diff
@@ -154,6 +148,10 @@ repos:
     hooks:
       - id: ruff
         files: python/.*$
+  - repo: https://github.com/rapidsai/pre-commit-hooks
+    rev: 253ff58a8a842a517995a7cdf6832aff20356b2a
```

Comment:
I'd like to adopt some kind of version/tag strategy for this repo, so we don't have to pin to hashes. https://github.com/rapidsai/pre-commit-hooks/issues/16

Reply:
I'd tag the current state of that repo as `v0.0.1` and move forward with that tag. Either way is fine but I suspect there will be a lot of other repos who will copy-paste this from cudf and we probably don't want them to use a hash.

Reply:
Created `v0.0.1` and updated to use it.